web/.eslintrc.js

```diff
@@ -0,0 +1,35 @@
+module.exports = {
+    'env': {
+        'browser': true,
+        'es2021': true,
+    },
+    'extends': [
+        // "react-app",
+        // 'plugin:react/recommended',
+        'google',
+        "plugin:prettier/recommended",
+        // "prettier/@typescript-eslint",
+        // "prettier/react"
+    ],
+    'parserOptions': {
+        'ecmaFeatures': {
+            'jsx': true,
+        },
+        'ecmaVersion': 13,
+        'sourceType': 'module',
+    },
+    'plugins': [
+        // 'react',
+    ],
+    'rules': {
```

Comment:
Why not just stick with eslint defaults?

Reply:
@saranshdhingra care to add here? I took these from your config figuring you had a reason for adding them.

Reply:
I think we're taking care of defaults by using the `eslint-config-google` package and declaring that we wish to use the `google` styleguide in the `extends` property.

Reply:
Alright. I have removed
```
"require-jsdoc": [
      "error",
      {
        require: {
          FunctionDeclaration: true,
          MethodDefinition: false,
          ClassDeclaration: false,
          ArrowFunctionExpression: false,
          FunctionExpression: false,
        },
      },
    ],
```